content/news/lpw-stepdown.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2020-07-22T00:00:00+00:00"
+title: "Lee-Ping Wang stepping down from the leadership role"
+tags: ["website","news"]
+categories: ["news"]
+draft: false
+description: "Lee-Ping Wang has decided to step down from his his role as an OpenFF primary investigator and to continue his involvement as a co-investigator."
+weight: 10
+author: "Karmen Condic-Jurkic"
+---
+
+The Open Force Field Initiative is sad to announce that [Lee-Ping Wang](https://chemistry.ucdavis.edu/people/lee-ping-wang) has decided to step down from his role as an OpenFF primary investigator, but we are very happy that Dr. Wang will stay actively involved as an OpenFF co-investigator role. The Open Force Field Initiative owes a great deal of its success to the work done by Dr. Wang and his lab members. His  [ForceBalance](https://github.com/leeping/forcebalance) software package sits at the heart of the OpenFF force field optimization infrastructure, and [torsiondrive](https://github.com/lpwgroup/torsiondrive), developed by Yudong Qiu, is used to generate QM torsion profiles datasets used in force field parameterization. But, his software contributions are only a small fraction of the time, effort and experience Dr. Wang has invested to build the Open Force Field Initiative and make this collaborative project so productive and successful. Dr. Wang is now seeking a way to create more time and space to explore some new and exciting research directions, which is the main motivation behind his decision to step down from the leadership role.
```

Comment:
Maybe not "is sad to announce"? "The Force Field Initiative" doesn't match the "we" after the comma---may as well just delete "The Open Force Field Initiative is sad to announce that" since the rest makes sense on its own?

Reply:
Sounds good to me and thanks for catching that typo!

Reply:
Modified to: 

> From August 1, the Open Force Field Initiative will undergo a change in the leadership team, as Lee-Ping Wang has decided to step down from his role as an OpenFF primary investigator. This not a farewell, but a change of roles and Dr. Wang will remain actively involved in OpenFF research efforts as a co-investigator.